### ISSUE TITLE
Ruby Gems version 2.5.1 has issues with undefined method "this"

### ISSUE THREAD

**Issue by remomueller** [remomueller/documentation#7]
If you see the following message:

```
/usr/local/rvm/rubies/ruby-2.3.0/lib/ruby/site_ruby/2.3.0/rubygems/specification.rb:2157:in `method_missing': undefined method `this' for #<Gem::Specification:0x3feeac908ac8 scss_lint-0.42.2> (NoMethodError)
	from /usr/local/rvm/rubies/ruby-2.3.0/lib/ruby/site_ruby/2.3.0/rubygems/specification.rb:1056:in `find_active_stub_by_path'
	from /usr/local/rvm/rubies/ruby-2.3.0/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:63:in `require'
```

Can be resolved by installing RubyGems 2.4.8

Using 2.4.8 fixes the issues with certain gems.

```
gem update --system 2.4.8 --no-ri --no-rdoc
```

Related to https://github.com/rubygems/rubygems/issues/1420

**Comment by jegodwin** [comment]
+1 on this. Thanks @remomueller; I've been dealing with it for a while and couldn't find a solution.

**Comment by remomueller** [comment]
Glad it helped! Thanks for the note on Rbenv. I just tried this again against RubyGems 2.6.1 but the issue still exists, and it also looks like the issue in the RubyGems repository is still open for the time being.

Mainly added this as a reminder to my future self in case I run into this again, and will probably add it to the [miscellaneous issues](https://github.com/remomueller/documentation/blob/master/macosx/199-miscellaneous.md) I've run into the past.

**Comment by jegodwin** [comment]
:+1:

**Comment by remomueller** [comment]
Looks like this is fixed with the RubyGems [2.6.3](https://rubygems.org/gems/rubygems-update/versions/2.6.3), closing issue.
